safe_transaction_service/history/tests/test_views_v2.py

```diff
@@ -461,7 +513,7 @@ def test_delegate_delete(self):
             )
             self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
             self.assertIn(
-                "Signature is not valid",
```

Comment:
This error message is no longer emitted. Since a signature can be valid with our without a Chain ID, we do not return early with this error.